src/Jobs/CheckLink.php

```diff
@@ -50,5 +51,13 @@ public function handle(): void
             $this->linkCheck->timeout = 0;
             $this->linkCheck->save();
         }
+
+        // Was this the last check in the batch?  If so, send a notification
+        if ($this->linkCheck->batchStatus($this->linkCheck->bastch_id) === 100) {
```

Comment:
batch_id

I think 100.00 is not going to be equal with 100? The underlying function uses division so a float might come up?